### PR TITLE
generate zsysconf_values_linux_loongarch64.go

### PR DESCRIPTION
**Pull request by merore**
I run  `go build . && ./mksysconf` generate zsysconf_values for linux_loongarch64. 

Currently golang of loongarch64 is waiting for the community to receive. All these values meets the definition in golang.org/x/sys/unix.  

I open this PR because [gopsutil](github.com/shirou/gopsutil) needs. I don’t know if this approach is reasonable, please contact me if  you have other suggestions.
```
root@merore-larch:/wk/github.com/merore/go-sysconf# file mksysconf
mksysconf: ELF 64-bit LSB executable, LoongArch-64, version 1 (SYSV), statically linked, Go BuildID=1zmjGuE9DKs8ISR2WaPz/eB5xeSi_POWE-l46-xb_/ZV6DwaJrPNtbVLEIOfCU/4b2Pcx90zjX8qISsaqb4, not stripped

root@merore-larch:/wk/github.com/merore/go-sysconf# lscpu 
Architecture:        loongarch64
Byte Order:          Little Endian
CPU(s):              4
On-line CPU(s) list: 0-3
Thread(s) per core:  1
Core(s) per socket:  4
Socket(s):           1
NUMA node(s):        1
Model name:          Loongson-3A5000LL
CPU max MHz:         2300.0000
CPU min MHz:         225.0000
BogoMIPS:            4592.64
L1d cache:           64K
L1i cache:           64K
L2 cache:            256K
L3 cache:            16384K
NUMA node0 CPU(s):   0-3

```

### CODE REVIEW

**Review by tklauser — COMMENTED** [review]
Thanks for the change @shanjiantao. I think it's fine adding support for Loongarch64 and the change itself looks OK. There currently is a proposal open for the addition of that architecture to Go mainline at golang/go#46229 and it seems the `GOARCH` value to use hasn't been decided yet.

I'll thus wait with merging this PR until the `GOARCH` value has been decided.

File: zsysconf_values_linux_loongarch64.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
+// Code generated by cmd/cgo -godefs; DO NOT EDIT.
+// cgo -godefs sysconf_values_linux.go
+
```
Comment (tklauser) [inline zsysconf_values_linux_loongarch64.go:3]: Also, we'll need proper `// +build` and `//go:build` lines here (depending on the `GOARCH` being chosen) in order to fix the build with Go versions which don't know about that yet.

Comment (shanjiantao) [reply zsysconf_values_linux_loongarch64.go:3]: good idea, `//+build` is a good way  at the moment. As for the `GOARCH` issue, just wait for their decision.

**Comment by tklauser** [comment]
It looks like the decision is `GOARCH=loong64`, see https://github.com/golang/go/issues/46229#issuecomment-861154674 The `GOARCH` value was added to `golang/go` in https://github.com/golang/go/commit/2b00a54baf2b677b2aaddd93c25b11ea4642a86f already.

@merore could you please update the PR to use `GOARCH=loong64` and also add the corresponding build tags for backwards compatibility? Then we should be good to merge. Thanks!

**Comment by xen0n** [comment]
Currently the kernel and libc ABIs are not upstream yet, thus subject to possible changes. The early version I saw was heavily MIPS-based, so has a very little chance to be merged as-is; and this PR contains values from that era, I think. So maybe this PR should wait until the ABIs are sufficiently settled?

**Comment by tklauser** [comment]
Thanks for the context @xen0n! I agree, in that case let's wait until the ABIs in kernel and libc are stable.

**Pull request closed without merging**